app/models/curry/unauthorized_commit_author_comment.rb

```diff
@@ -44,7 +44,7 @@ def comment_body
         verified email address. To become authorized to contribute, you will
         need to sign the Contributor License Agreement (CLA) as an individual or
         on behalf of your company. [You can read more on Chef's
-        blog.](#{chef_blog_url('2014/06/23/changes-to-the-contributor-license-agreement-process')})
+        blog.](https://www.chef.io/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)
```

Comment:
Thought about this long and hard...I think the risk is a little too great to move the helper methods into lib for this breakfix.  For now, I will include CustomUrlHelper in the model (we're already doing this with the application controller), but will be creating a ticket to refactor the helper into lib in the future.

Just implemented this and pushed to this branch, this is now ready for review again.